bin/configs/csharp-generichost-net8-oneOf-3.1.0.yaml

```diff
@@ -0,0 +1,14 @@
+# for csharp generichost
+generatorName: csharp
+outputDir: samples/client/petstore/csharp/generichost/net8/OneOfDiscriminator-3.1.0
```

Comment:
thanks for the PR, which looks good me.

after merging, i'll replace this test (sample) with a unit test instead similar to other 3.1 tests we've added so far.

Reply:
done via https://github.com/OpenAPITools/openapi-generator/pull/18828